phoenix-ios/phoenix-ios/prefs/UserDefaults+Codable.swift

```diff
@@ -118,14 +134,20 @@ struct LiquidityPolicy: Equatable, Codable {
 		return skipAbsoluteFeeCheck ?? NodeParamsManager.companion.defaultLiquidityPolicy.skipAbsoluteFeeCheck
 	}
 	
+	var effectiveMaxAllowedFeeCredit: Lightning_kmpMilliSatoshi {
+		return NodeParamsManager.companion.defaultLiquidityPolicy.maxAllowedFeeCredit
+	}
+	
 	func toKotlin() -> Lightning_kmpLiquidityPolicy {
 		
 		if enabled {
 			
 			return Lightning_kmpLiquidityPolicy.Auto(
-				maxAbsoluteFee: Bitcoin_kmpSatoshi(sat: effectiveMaxFeeSats),
+				inboundLiquidityTarget: effectiveInboundLiquidityTarget,
+				maxAbsoluteFee: effectiveMaxFee,
 				maxRelativeFeeBasisPoints: effectiveMaxFeeBasisPoints,
-				skipAbsoluteFeeCheck: effectiveSkipAbsoluteFeeCheck
+				skipAbsoluteFeeCheck: effectiveSkipAbsoluteFeeCheck,
+				maxAllowedFeeCredit: effectiveMaxAllowedFeeCredit
```

Comment:
@robbiehanson Just to be sure this is the only place where the iOS app defines the liquidity policy ?

Reply:
Yes, the iOS app should be pulling all default values directly from NodeParamsManager